schemas/2019-04-01/autogeneratedResources.json

```diff
@@ -261,6 +261,9 @@
         },
         {
           "$ref": "https://schema.management.azure.com/schemas/2019-04-01/Microsoft.DevSpaces.json#/resourceDefinitions/controllers"
+        },
+        {
+          "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Insights.json#/resourceDefinitions/actionGroups"
```

Comment:
Please can you also manually this resource to the following files - this process has not yet been automated.
https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2014-04-01-preview/deploymentTemplate.json
https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2015-01-01/deploymentTemplate.json

Reply:
@antmarti-microsoft  . I have completed the changes as requested. pls approve